### PR TITLE
Docs2 controls & source

### PR DESCRIPTION
**Pull request by shilman**
Issue: N/A

## What I did

- [x] Got source working in Docs2
- [x] Got controls working in Docs2

## How to test

```
yarn bootstrap --core
cd examples/external-docs
yarn storybook
yarn dev
open http://localhost:3000
```

### CODE REVIEW

File: lib/components/src/Zoom/ZoomElement.tsx [hunk 1]
```diff
@@ -10,7 +10,6 @@ const ZoomElementWrapper = styled.div<{ scale: number; height: number }>(({ scal
         },
       }
     : {
-        height: height + 50,
```
Comment (yannbf) [inline lib/components/src/Zoom/ZoomElement.tsx:13]: Why was this change necessary?

**Comment by ndelangen** [comment]
@shilman feel free to open back up again; but I assume this PR is not going to be merged.
